batch/batches/LiveToVod/KAsyncLiveToVod.class.php

```diff
@@ -31,13 +31,24 @@ protected function getJobType()
 	{
 		return KalturaBatchJobType::LIVE_TO_VOD;
 	}
-	
+
 	/* (non-PHPdoc)
 	 * @see KJobHandlerWorker::exec()
 	 */
 	protected function exec(KalturaBatchJob $job)
 	{
-		return $this->copyCuePoint($job, $job->data);
+		$jobRes = $this->copyCuePoint($job, $job->data);
+
+		$liveEntryId = $job->data->liveEntryId;
+		$liveEntry = KBatchBase::$kClient->baseEntry->get($liveEntryId);
```

Comment:
if for some reason this API call fails it will throw an exception, make sure to catch it.